src/renderer/utils/deleteNoteIfConfirmed.ts

```diff
@@ -11,7 +11,11 @@ export async function deleteNoteIfConfirmed(
   const { notes } = ctx.getState();
 
   const note = getNoteById(notes, noteId);
-  const confirmed = await promptConfirmAction("delete", `note ${note.name}`);
+  const confirmed = await promptConfirmAction(
+    "delete",
+    `"${note.name}"`,
+    "The note can be restored from your computer's trash",
+  );
```

Comment:
This was changed to be a little easier to read. Previously it'd say "delete note Foo" but now it'll say "delete "foo"".